### PR TITLE
Upgrade notes for palette 4.0

### PR DESCRIPTION
**Pull request by ritawatson**
## Describe the Change

This PR provides upgrade notes to upgrade Palette from 3.x to 4.0 and adds a new doc with steps to upload certs to the Palette console.

## Review Changes

💻 
[4.0 Upgrade Notes 4.0](https://deploy-preview-1477--docs-spectrocloud.netlify.app/enterprise-version/upgrade#palette4.0)
[Upload SSL Certificate](https://deploy-preview-1477--docs-spectrocloud.netlify.app/enterprise-version/ssl-certificate-management)

🎫 [Jira Ticket](https://spectrocloud.atlassian.net/browse/DOC-840)

### CODE REVIEW

File: content/docs/12-enterprise-version/10-upgrade.md [hunk 1]
```diff
@@ -13,6 +13,81 @@ fullWidth: false
...
+  <br />
+
+  If you already have an SSL certificate, key, and Certificate Authority (CA) certificate, you can use them when upgrading to Palette 4.0.0. To learn how to add a certificate to Palette, refer to [Add Certificate](/enterprise-version/upgrade#addcertificate).
+
+
+- **Upgraded on-prem Palette Kubernetes** - the Kubernetes version used for on-prem Palette is upgraded from 1.24 to 1.25. You will need to copy the new Kubernetes YAML to the Kubernetes layer in the Enterprise cluster profile. If you have customized your Kubernetes configuration, you will need to manually adjust custom values and include any additional configuration in the upgraded YAML that we provide. Refer to [Upgrade Kubernetes](/enterprise-version/upgrade#upgradekubernetes.)
+
+## Upgrade from Palette 3.x to 4.0
+
+From the Enterprise cluster system console, click the **Update version** button. Palette will be temporarily unavailable while system services update.
```
Comment (karl-cardenas-coding) [inline content/docs/12-enterprise-version/10-upgrade.md:33]:
```suggestion
From the Palette system console, click the **Update version** button. Palette will be temporarily unavailable while system services update.
```
I think we should just say Palette system console. The term enterprise cluster can be a bit confusing.

File: content/docs/12-enterprise-version/10-upgrade.md [hunk 2]
```diff
@@ -13,6 +13,81 @@ fullWidth: false
...
+<br />
+
+- **Enhanced security for Palette microservices** - To enhance security, all microservices within Palette now use `insecure-skip-tls-verify` set to `false`. When upgrading to Palette 4.0, you must provide a valid SSL certificate in the system console. 
+
+  <br />
+
+  If you already have an SSL certificate, key, and Certificate Authority (CA) certificate, you can use them when upgrading to Palette 4.0.0. To learn how to add a certificate to Palette, refer to [Add Certificate](/enterprise-version/upgrade#addcertificate).
+
+
+- **Upgraded on-prem Palette Kubernetes** - the Kubernetes version used for on-prem Palette is upgraded from 1.24 to 1.25. You will need to copy the new Kubernetes YAML to the Kubernetes layer in the Enterprise cluster profile. If you have customized your Kubernetes configuration, you will need to manually adjust custom values and include any additional configuration in the upgraded YAML that we provide. Refer to [Upgrade Kubernetes](/enterprise-version/upgrade#upgradekubernetes.)
```
Comment (karl-cardenas-coding) [inline content/docs/12-enterprise-version/10-upgrade.md:29]:
```suggestion
- **Self-hosted Palette Kubernetes Upgrade** - If you installed Palette using the Helm Chart method, the Kubernetes version used for Palette is upgraded from 1.24 to 1.25. You will need to copy the new Kubernetes YAML to the Kubernetes layer in the Enterprise cluster profile. If you have customized your Kubernetes configuration, you will need to manually adjust custom values and include any additional configuration in the upgraded YAML that we provide. Refer to [Upgrade Kubernetes](/enterprise-version/upgrade#upgradekubernetes.)
```
A bit more additional context.

File: content/docs/12-enterprise-version/10-upgrade.md [hunk 3]
```diff
@@ -13,6 +13,77 @@ fullWidth: false
...
+<br />
+
+- **Enhanced security for Palette microservices** - To enhance security, all microservices within Palette now use `insecure-skip-tls-verify` set to `false`. When upgrading to Palette 4.0, you must provide a certificate in the system console. 
+
+  <br />
+
+  If you already have a configured certificate, key, and Certificate Authority (CA) certificate, you can use them when upgrading to Palette 4.0. To learn how to add a certificate to Palette, refer to [Add Certificate](/enterprise-version/upgrade#addcertificate).
+
+
+- **Upgraded on-prem Palette Kubernetes** - the Kubernetes version used for on-prem Palette is upgraded from 1.24 to 1.25. You will need to copy the new Kubernetes YAML to the Kubernetes layer in the Enterprise cluster profile. If you have customized your Kubernetes configuration, you will need to manually adjust custom values and include any additional configuration in the upgraded YAML that we provide. Refer to [Upgrade Kubernetes](/enterprise-version/upgrade#upgradekubernetes.)
```
[the remark this answers is not in the document]
Comment (ritawatson) [reply content/docs/12-enterprise-version/10-upgrade.md:29]: added

File: content/docs/12-enterprise-version/10-upgrade.md (hunk 2 above)
Comment (ritawatson) [reply content/docs/12-enterprise-version/10-upgrade.md:29]: added

File: content/docs/12-enterprise-version/10-upgrade.md [hunk 4]
```diff
@@ -13,6 +13,77 @@ fullWidth: false
...
+
+## Upgrade from Palette 3.x to 4.0
+
+From the Enterprise cluster system console, click the **Update version** button. Palette will be temporarily unavailable while system services update.
+
+![Screenshot of the "Update version" button in the system consoles.](/enterprise-version_sys-console-update-palette-version.png)
+
+<br />
+
+## Add Certificate
```
[the remark this answers is not in the document]
Comment (ritawatson) [reply content/docs/12-enterprise-version/10-upgrade.md]: Created a copy of the VerteX SSL cert management doc, removed VerteX, and added Palette-flavored console screenshot.

**Review by karl-cardenas-coding — APPROVED** [review]

**Pull request merged**